configurations/off.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
```

Comment:
> As no eslint rules are enabled by default, when would someone need to import this?

If you do not import all the off rules then `eslint-find-rules` would fail with the `-u` option